creator-node/src/routes/nodeSync.js

```diff
@@ -98,14 +105,20 @@ module.exports = function (app) {
         cnodeUser['clockRecords'] = []
 
         cnodeUsersDict[cnodeUser.cnodeUserUUID] = cnodeUser
+        const curCnodeUserClockVal = cnodeUser.clock
 
-        // TODO - remove this once we no longer have a MaxClock in export
-        // this just overrides the clock value to the max clock we're sending over to the secondary so it knows
-        // there's more data to pull
-        if (cnodeUser.clock > MaxClock) {
+        // Resets cnodeUser clock value to requestedClockRangeMax to ensure consistency with clockRecords data
+        // Also ensures secondary knows there is more data to sync
+        if (cnodeUser.clock > requestedClockRangeMax) {
```

Comment:
is this correct - this data is exported into the cnodeUser object which is sent over the wire to the secondary hence ensuring that highest clock is not greater than the requestedClockRangeMax but has no effect on the _actual_ clock value stored in primary DB?

Reply:
resolved per offline discussion